BoneCentral/WebApplication/messenger.js

```diff
@@ -0,0 +1,36 @@
+var hdwr = require('./hardware.js');
```

Comment:
I felt the same. There were some occassions in other projects where this wasn't the case for me. Feel free to refactor it out.